source/backend/functions/cost-parser/package.json

```diff
@@ -23,7 +23,7 @@
     "ramda": "0.28.0"
   },
   "devDependencies": {
-    "aws-sdk": "latest",
+    "aws-sdk": "2.1354.0",
```

Comment:
why is this dev dependency? are we not packaging this with the lambda binary?

Reply:
that's good, but should this not be a **prod** dependency and packaged with the lambda binary?

Reply:
This needs to be re-evaluated by the team. We don't control sdk versions on the lambda, this can potentially result in unexpected behavior. We should be packaging version that we have tested against during development for deterministic behavior in customer environment.